app/src/main/res/values/strings.xml

```diff
@@ -199,6 +199,7 @@
     <string name="cancel_current_timer">Cancel current timer</string>
     <string name="sleep_timer_canceled">Sleep timer canceled.</string>
     <string name="sleep_timer_set">Sleep timer set for %d minutes from now.</string>
+    <string name="time_unit_min">min</string>
```

Comment:
Please use the full word "minute" to avoid misunderstanding.

Reply:
I left it as "min" because that's what it said there before.

Reply:
I mean to rename `time_unit_min` to `time_unit_minute`.